fpage/views.py

```diff
@@ -19,6 +21,48 @@
   #  model = Album
    # template_name='fpage/detail.html'
 
+def registerPage(request):
+    if request.user.is_authenticated:
+        return redirect('fpage:index')
+    else:
+        form = CreateUserForm()
+        if request.method == 'POST':
+            form = CreateUserForm(request.POST)
+            if form.is_valid():
+                form.save()
+                user = form.cleaned_data.get('username')
+                messages.success(request, 'Account was created for ' + user)
+
+                return redirect('fpage:login')
+			
+
```

Comment:
Add an else statement with logic to handle register form validation problems. Try to show the user specific error msgs like username already exists, password don't match.

Reply:
@ExpressHermes The condition is already sastisfied if a user exists it shows "A user with that username already exists." and if the password doesnt match it also shows a message.

Reply:
Okay. I just saw that logic. It's good.